docs/data/base/components/input/input.md

```diff
@@ -54,27 +54,17 @@ The Input component is composed of a root `<div>` slot that houses one interior
 </div>
 ```
 
-### Slot props
+### Custom structure
 
-:::info
-The following props are available on all non-utility Base components.
-See [Usage](/base/getting-started/usage/) for full details.
-:::
-
-Use the `component` prop to override the root slot with a custom element:
-
-```jsx
-<Input component="aside" />
-```
-
-Use the `slots` prop to override any interior slots in addition to the root:
+Use the `slots` prop to override the root or any other interior slot:
 
 ```jsx
-<Input slots={{ root: 'aside' }} />
+<Input slots={{ root: 'aside', input: 'button' }} />
```

Comment:
`input: 'button'` likely won't work well (and it doesn't have sense). 

```suggestion
<Input slots={{ root: 'aside', input: CustomInput }} />
```

Reply:
addressed